pyne/material.pyx

```diff
@@ -1813,6 +1858,66 @@ def from_atom_frac(atom_fracs, double mass=-1.0, double density=-1.0,
         mat.atoms_per_molecule = atoms_per_molecule
 
     return mat
+    
+
+
+def from_activity(activities, double mass=-1.0, double density=-1.0,
+                   double atoms_per_molecule=-1.0, metadata=None):
+    """from_activity(activities, double mass=-1.0, double atoms_per_molecule=-1.0)
+    Create a Material from a mapping of radionuclide activities.
+
+    Parameters
+    ----------
+    activities : dict
```

Comment:
units are only important here in the case where the mass is not provided... not sure how to document that.